internal/storage/payments.go

```diff
@@ -323,6 +326,35 @@ func (s *store) PaymentsDeleteFromAccountID(ctx context.Context, accountID model
 	return e("failed to delete payments", err)
 }
 
+func (s *store) PaymentsDeleteFromPSUID(ctx context.Context, psuID uuid.UUID) error {
+	_, err := s.db.NewDelete().
+		Model((*payment)(nil)).
```

Comment:
```suggestion
		Table("payments").
```

Your way is fine too, but since you don't expect a return value you can also just do this.